From bcc0172b68244fd431934f1a2312c0feb3659403 Mon Sep 17 00:00:00 2001
From: Yintian Tao <yttao@amd.com>
Date: Thu, 4 Jan 2018 16:58:16 +0800
Subject: [PATCH 4573/5855] drm/amd/powerplay: fix typo error for '3be7be08ac'

Due to typo error, it will cause compile error so fix it.
Change-Id: Iabe7158e08e6aef155ca3394cafc6eb4256a0030
Signed-off-by: Yintian Tao <yttao@amd.com>
Reviewed-by: Rex Zhu <Rex.Zhu@amd.com>
---
 drivers/gpu/drm/amd/powerplay/smumgr/smu7_smumgr.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/powerplay/smumgr/smu7_smumgr.c b/drivers/gpu/drm/amd/powerplay/smumgr/smu7_smumgr.c
index 7dc4cee..25dd778 100644
--- a/drivers/gpu/drm/amd/powerplay/smumgr/smu7_smumgr.c
+++ b/drivers/gpu/drm/amd/powerplay/smumgr/smu7_smumgr.c
@@ -648,7 +648,7 @@ int smu7_init(struct pp_hwmgr *hwmgr)
 
 int smu7_smu_fini(struct pp_hwmgr *hwmgr)
 {
-	struct smu7_smumgr smu_data = (struct smu7_smumgr *)(hwmgr->smu_backend);
+	struct smu7_smumgr *smu_data = (struct smu7_smumgr *)(hwmgr->smu_backend);
 
 	smu_free_memory(hwmgr->device, smu_data->header_buffer.handle);
 	if (!cgs_is_virtualization_enabled(hwmgr->device))
-- 
2.7.4

